tests/drum/test_fit.py

```diff
@@ -119,6 +118,69 @@ def test_fit(
             cmd, "Failed in {} command line! {}".format(ArgumentsOptions.MAIN_COMMAND, cmd)
         )
 
+    @pytest.mark.parametrize(
+        "framework", [RDS, SKLEARN_BINARY, SKLEARN_REGRESSION, XGB, KERAS, PYTORCH]
+    )
+    @pytest.mark.parametrize("problem", [BINARY_TEXT, REGRESSION, ANOMALY])
+    @pytest.mark.parametrize("docker", [DOCKER_PYTHON_SKLEARN, None])
+    @pytest.mark.parametrize("weights", [WEIGHTS_CSV, WEIGHTS_ARGS, None])
+    def test_fit(
```

Comment:
???

Reply:
I added  `test_fit_for_use_output_and_nested` which tests only nesting and output dir with one ML problem, thus i removed these params permutations from `test_fit` which significantly reduces number of test cases.